classes/journal/categories/CategoryDAO.inc.php

```diff
@@ -36,13 +36,19 @@ function getCategories() {
 	}
 
 	/**
-	 * Rebuild the cache.
+	 * Get an iterator of category objects.
 	 */
-	function rebuildCache() {
-		// Read the full set of categories into an associative array
+	function getIterator() {
```

Comment:
Also, it's used anywhere else outside the class? If not, we can hide it declaring it protected